baybe/parameters/enum.py

```diff
@@ -17,17 +17,60 @@ class CategoricalEncoding(ParameterEncoding):
     """Integer encoding."""
 
 
+class SubstanceEncodingAliases(ParameterEncoding):
+    """Aliases for SubstanceEncoding values."""
+
+    MORGAN_FP = "ECFP"
+
+
 class SubstanceEncoding(ParameterEncoding):
     """Available encodings for substance parameters."""
 
-    MORDRED = "MORDRED"
-    """Encoding based on Mordred chemical descriptors."""
+    AtomPairFingerprint = "ATOMPAIR"
```

Comment:
Usually, we use UPPER_CASE writing for enum values (see other encodings, for example). Also, not sure if the "Fingerprint" at the end is really necessary? If possible, I'd rather sync the enum field name with its value

Reply:
We wanted to map parameters (currently) values to class names used for `scikit-fingerprint` (names). I can probably switch the two (but need to test if that breaks sth else).

Reply:
Now the SubstanceEncoding enum follows this rule, but there is a new enum with mapping to skfp classes (added for the new helper function that converts fp names to cls+params